setup.py

```diff
@@ -94,7 +90,7 @@ def find_files(pattern, path=None, root="rez"):
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Topic :: Software Development",
-        "Topic :: System :: Software Distribution"
+        "Topic :: System :: Software Distribution",
     ],
     python_requires=">=3.7",
```

Comment:
Not that it's part of your PR but maybe we should bound this to <3.14, so that when 3.14 comes around and people try to install it, they get an actual error, and then we actually go and add support, rather than everyone putting in issues that it doesn't work for <whatever the reason was that time>